universal-application-tool-0.0.1/app/services/MessageKey.java

```diff
@@ -2,22 +2,53 @@
 
 /** Contains keys into the {@code messages} files used for translation. */
 public enum MessageKey {
-  // TODO(cdanzi): Add all message keys to this file.
+  ADDRESS_LINE_2_LABEL("label.addressLine2"),
```

Comment:
(really nonblocking) I feel like these should closely match the actual string they hold, so people can find them more easily.

e.g.`LABEL_ADDRESS_LINE2` or `TOAST_APPLICATION_SAVED`.

Reply:
good point! I updated the keys to hopefully be more similar to the `messages` file